.github/workflows/security.yml

```
@@ -0,0 +1,107 @@
+        with:
+          category: "/language:javascript"
+          
+      # Snyk security scan
+      - name: Run Snyk to check for vulnerabilities
+        uses: snyk/actions/node@master
+        continue-on-error: true
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+        with:
+          args: --severity-threshold=high
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)